### PR TITLE
N data interestingness

### PR DESCRIPTION
**Pull request by Delfil**
Computes interestingness, decreasing/increasing it when the percentage of N's within the DataNodes of a wrapper is high.

### CODE REVIEW

File: src/main/java/tudelft/ti2806/pl3/visualization/FilteredGraphModel.java [hunk 1]
```diff
@@ -95,6 +96,7 @@ public void produceWrappedGraphData() {
 		positionNodeYOnGenomeSpace.calculate(collapsedNode, null);
 		// sizeInterest.calculate(collapsedNode, null);
 		calculateCollapse.compute(collapsedNode);
+		new CalculateNInterest(-1000000000).calculate(this.collapsedNode, null);
```
Comment (mathieupost) [inline src/main/java/tudelft/ti2806/pl3/visualization/FilteredGraphModel.java]: Where does the -1000000000 come from?

File: src/main/java/tudelft/ti2806/pl3/data/wrapper/operation/interest/CalculateNInterest.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+	 * interesting.
+	 */
+	private final int interest;
+	
+	public CalculateNInterest(int interest) {
+		this.interest = interest;
+	}
+	
+	@Override
+	public void calculate(DataNodeWrapper wrapper, Wrapper container) {
```
Comment (borismattijssen) [inline src/main/java/tudelft/ti2806/pl3/data/wrapper/operation/interest/CalculateNInterest.java]: You only hanlde the case of DataNodeWrapper, but these are only used from the Unwrap operation and after that. So should the call to CalculateNInterest not be in ZoombedGraphModel?

File: src/main/java/tudelft/ti2806/pl3/visualization/FilteredGraphModel.java (hunk 1 above)
Comment (kwendel) [inline src/main/java/tudelft/ti2806/pl3/visualization/FilteredGraphModel.java]: Do not use magic numbers

**Comment by kwendel** [comment]
Looks good now!

**Comment by borismattijssen** [comment]
Yeh all good!

**Pull request merged**